.travis.yml

```diff
@@ -1,3 +1,37 @@
+
+Skip to content
+Pull requests
+Issues
+Marketplace
+Explore
+@puigmontella
+SedFoam /
+sedfoam
+
+11
+28
+
+    21
+
+Code
+Issues
+Pull requests
+Actions
+Projects
+Wiki
+Security
+
+    Insights
+
+sedfoam/.travis.yml
+@CyrilleBonamy
+CyrilleBonamy new css doc (#33)
+Latest commit 1ad0c8e 18 days ago
+History
+2 contributors
+@CyrilleBonamy
+@jchauchat
+100 lines (95 sloc) 3.5 KB
```

Comment:
What is that?
I think you can delete the 34 first lines.

Reply:
ok! I don't know how I messed it up!